### PR TITLE
fix(links): upgrade & migrate cards' links

### LINKED ISSUE #2211: [Bug] Link to Migrate from Tauri 1 page is broken
### Question you want answered

How to migrate from v1 to v2

### Where did you look for an answer?

.

### Page URL

https://v2.tauri.app/start/migrate/

### Additional context

The link that redirects to _Migrate from Tauri 1_ page is broken in the above mentioned page.

### Are you willing to work on this yourself?

- [X] I want to work on this myself

### PR DESCRIPTION
**Pull request by kareemmahlees**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- Closes #2211 
- What does this PR change?:
  - Adds trailing slash for the links 1eb73e663b7f28777db673e497ff435547f287ea
  - Adds missing card for `Upgrade Existing Project` 4411bb2abc359201c5fca774ca47ba436ea68738
- Did you change something visual? No

### CODE REVIEW

**Comment by kareemmahlees** [comment]
I also noticed that in the `zh-cn` translation, the links are completely broken and redirect to 404 pages.
Is this intentional? if not, I can fix those along the way too.

**Comment by vasfvitor** [comment]
> I also noticed that in the `zh-cn` translation, the links are completely broken and redirect to 404 pages. Is this intentional? if not, I can fix those along the way too.

it is not, we had a massive docs reestructure and many stuff broke in the process, i18n mostly

would you open another PR to fix these? or maybe here it's ok if it's not that much links

File: src/content/docs/start/migrate/index.mdx [hunk 1]
```diff
@@ -12,17 +12,22 @@ import { LinkCard, CardGrid } from '@astrojs/starlight/components';
...
     title="Migrate from Flutter"
-    href="/start/migrate/from-flutter"
+    href="/start/migrate/from-flutter/"
     description="Read more about general tips and tricks for migrating a Flutter application to Tauri."
   />
+  <LinkCard
+    title="Upgrade Existing Project"
+    href="/start/migrate/from-existing/"
+    description="Read more about general tips and tricks for upgrading your existing Tauri app."
+  />
```
Comment (vasfvitor) [inline src/content/docs/start/migrate/index.mdx:32]: Great!

**Review by vasfvitor — APPROVED** [review]

**Comment by vasfvitor** [comment]
thank you

**Pull request merged**